src/commands/detail.js

```diff
@@ -21,8 +21,11 @@ module.exports = async function detail(flags) {
 	if (flags.focus?.length === 1 && flags.focus[0] === 'templates') {
 		filter = (repo) => repo.isTemplate;
 	}
+	if (flags.json) {
+		ls(flags);
+		return null;
+	}
 
-	// Get all repositories
```

Comment:
let's try to do the same reworking where the business logic is in one place, and the output logic is here.